src/Psalm/Internal/Analyzer/ClassLikeAnalyzer.php

```diff
@@ -331,6 +332,28 @@ public static function checkFullyQualifiedClassLikeName(
             return null;
         }
 
+        foreach ($class_storage->invalid_inheritors as $dependency_class_name => $_) {
+            IssueBuffer::maybeAdd(
+                new InvalidExtendClass(
```

Comment:
Unsure on the process for using/reusing Issue classes. Might need a new one for this?

Reply:
The issue is that this specific error is used to prevent inheritance of final classes. This is a big issue so it will be reported at a very high level (in fact, it will be reported on any level). Your issue however will not provoke any crash so you don't want to report it that soon. So another issue would be better here

Reply:
Yeah :)